templates/template_index.html

```diff
@@ -206,18 +206,78 @@ <h2 class="mb-4 text-center">Compatibility</h2>
   }
 }
 
+var titlefilter = null;
+var statusfilter = null;
+var hiddenstatuses = []; 
+
 // Simple filter via: https://stackoverflow.com/a/44736154
 $("#filter").keyup(delay(function() {
-  var filter = $(this).val();
+  titlefilter = $(this).val();
+  filtertitles();
+}, 250));
+
+function filterstatus(status)
+{
+  if(statusfilter != null && statusfilter === status)
```

Comment:
Minor formatting nits:
- Use space between `if` and open paren `(`: `if (`
- Use open/close brace on same line as `if` and `else`
  ```
  if (...) {
    ...
  } else {
    ...
  }
  ```
- Space between comment token and first character of comment: `// User has ...`
- Use underscore to separate words in an identifier `status_to_hide` or `hidden_status_list` etc